python/ray/tests/accelerators/test_amd_gpu1.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
we can merge this with the existing `test_amd_gpu.py`

Reply:
Add the tests in `test_amd_gpu1.py` to `test_amd_gpu.py`. I think some tests can be combined with `pytest.param()`